.npmignore

```diff
@@ -1,13 +0,0 @@
-/.idea/
-/node_modules/
-/lib/tiny-lru.min.js
-/lib/tiny-lru.min.js.map
-/test/
-/src/
-.git
-.gitignore
-.eslintrc
-.travis.yml
-Gruntfile.js
-benchmark.js
-bower.json
```

Comment:
It's better to use whitelist as opposed to maintaining whitelist to ignore files during publish. See https://gist.github.com/ceejbot/610773f9990a751db68dc6c66974b8ec#whitelisting-files-with-the-files-array

Reply:
This doesn't work well IRL, as it'll include a CHANGELOG if present

Reply:
nm, it's always included 👍